contracts/CO30_Andrew_Outside.Tanoa/init.sqf

```diff
@@ -0,0 +1,89 @@
+[
+    laptop1,
+    1,  // texture source, (in the attributes of the object, Texture #X)
+    50, // size in GB
+    10,  // time to upload in seconds
+    {
+        systemChat "Server: Upload Started";
+    },
+    {
+        systemChat "Server: Upload Finished";
+        // Clear the screen after 2 seconds
+        [{
+            params ["_object"];
+            _object setObjectTextureGlobal [1, ""];
+        }, _this, 2] call CBA_fnc_waitAndExecute;
+    }
```

Comment:
This is just for example purposes in the code documentation, you can remove it from here, we don't want system chats in our missions

Reply:
You could also increase the time, 10 seconds is a little low if you want your QRFs to hit them while they do it

Reply:
I will attempt to remove the code then.

I will set forth a longer time as well.

Reply:
> systemChat "Server: Upload Started";
    },
    {
        systemChat "Server: Upload Finished";
       

is this the only part of that code I am removing?

Reply:
> 
    {
        systemChat "Server: Upload Started";
    },
    {
        systemChat "Server: Upload Finished";
        // Clear the screen after 2 seconds
        [{
            params ["_object"];
            _object setObjectTextureGlobal [1, ""];
        }, _this, 2] call CBA_fnc_waitAndExecute;
    }

I have now realized you put the whole thing as that is what you'd want me to remove. I will remove the entirety. Thanks.